Libraries/src/Amazon.Lambda.KinesisAnalyticsEvents/KinesisAnalyticsOutputDeliveryEvent.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+
+namespace Amazon.Lambda.KinesisAnalyticsEvents
+{
+    /// <summary>
+    /// This class represents the event from Kinesis Analytics application output.
+    /// </summary>
+    [DataContract]
+    public class KinesisAnalyticsOutputDeliveryEvent
+    {
+
+        /// <summary>
+        /// Gets or sets the invocation identifier.
+        /// </summary>
+        /// <value>
+        /// The invocation identifier.
+        /// </value>
+        [DataMember(Name = "invocationId")]
+        public string InvocationId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the application arn.
+        /// </summary>
+        /// <value>
+        /// The application arn.
+        /// </value>
+        [DataMember(Name = "applicationArn")]
+        public string ApplicationArn { get; set; }
+
+        /// <summary>
+        /// Gets or sets the records.
+        /// </summary>
+        /// <value>
+        /// The records.
+        /// </value>
+        [DataMember(Name = "records")]
+        public IList<DeliveryRecord> Records { get; set; }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        [DataContract]
+        public class DeliveryRecord
+        {
+            /// <summary>
+            /// Gets or sets the record identifier.
+            /// </summary>
+            /// <value>
+            /// The record identifier.
+            /// </value>
+            [DataMember(Name = "recordId")]
+            public string RecordId { get; set; }
+
+            /// <summary>
+            /// Gets or sets the record metadata.
+            /// </summary>
+            /// <value>
+            /// The record metadata.
+            /// </value>
+            [DataMember(Name = "lambdaDeliveryRecordMetadata")]
+            public LambdaDeliveryRecordMetadata RecordMetadata { get; set; }
+
+            /// <summary>
+            /// 
+            /// </summary>
+            [DataContract]
+            public class LambdaDeliveryRecordMetadata
+            {
+                /// <summary>
+                /// Gets or sets the retry hint.
```

Comment:
Can we add documentation that says retry hints are the number of delivery retries. Otherwise it is a confusing property.

Reply:
Addressed.